fern/api/openapi/openapi.yaml

```diff
@@ -3259,7 +3259,7 @@ paths:
       - present-proof v2.0
   /present-proof-2.0/records/{pres_ex_id}/credentials:
     get:
-      operationId: get_pres_ex_credentials
+      operationId: get_pres_ex_credentials-v2
```

Comment:
operationIds in OpenAPI are globally unique and our OpenAPI importer assumes this to be true